dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/AbstractTrackerPersister.java

```diff
@@ -209,11 +224,12 @@ public TrackerTypeReport persist(EntityManager entityManager, TrackerBundle bund
   protected abstract void persistOwnership(TrackerBundle bundle, T trackerDto, V entity);
 
   /** Execute the persistence of Data values linked to the entity being processed */
-  protected abstract void updateDataValues(
+  protected abstract void handleEventValueChanges(
```

Comment:
Can we make this concrete with a noop implementation and then remove the noop implementations from all but event persisters?

Reply:
that was my original idea: https://github.com/dhis2/dhis2-core/pull/19007#discussion_r1843948433